packages/react-storage/src/components/StorageBrowser/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -9,7 +11,11 @@ interface ErrorBoundaryState {
 }
 
 const Fallback = (): React.JSX.Element => (
-  <p>Something went wrong. Please try again.</p>
+  <div className={CLASS_BASE}>
+    <div className={`${CLASS_BASE}__error`}>
+      Something went wrong. Please try again.
```

Comment:
Since there's nothing actionable here, let's remove "Please try again."

Reply:
Good point, updated in latest.